common/lib/mandrill_client/client.py

```diff
@@ -38,40 +46,66 @@ class MandrillClient(object):
     REFERRAL_FOLLOW_UP_EMAIL = 'referred-learner-follow-up'
     REFERRAL_SOCIAL_IMPACT_TOOLKIT = 'social-impact-toolkit'
     MINI_COURSE_ENROLMENT = 'mini-course-enrolment'
+    SEND_ACTION_PLAN = 'send-action-plan'
 
     def __init__(self):
         self.mandrill_client = mandrill.Mandrill(settings.MANDRILL_API_KEY)
 
-    def send_mail(self, template_name, user_email, context, attachments=[], subject=None):
+    def get_email_to(self, receiver_emails_string):
+        """
+        parsing the comma separated email to a list of emails
+
+        Arguments:
+            receiver_emails_string: String contains a comma separated emails
+
+        Returns:
+            receiver_emails (List): A List of receiver email
+        """
+        email_list = receiver_emails_string.split(',')
+        receiver_emails = [{'email': email} for email in email_list]
+        return receiver_emails
+
+    def send_mail(self, template_name, receiver_emails_string, context, attachments=None, subject=None,
+                  reply_to_email=None, images=None):
         """
         calls the mandrill API for the specific template and email
 
         arguments:
         template_name: the slug/identifier of the mandrill email template
-        user_email: the email of the receiver
+        user_email: the email or comma separated emails of the receiver's
         context: the data which is passed to the template. must be a dict
+        reply_to_email:  email for reply_to
+        images: images attachments for referring it from content of email template
         """
+        if images is None:
+            images = []
+        if attachments is None:
+            attachments = []
```

Comment:
these can be simplified more
```
images = images or []
attachments = attachments or []
```

Reply:
I think the if statement is more readable then this expression. that why we have used it earlier too

Reply:
Code is always read by a developer and the suggested approach is readable as well as simpler.. Please update .

Reply:
Done